module/VuFind/src/VuFind/IdentifierLinker/UnpaywallFactory.php

```diff
@@ -69,7 +69,7 @@ public function __invoke(
         if (!empty($options)) {
             throw new \Exception('Unexpected options passed to factory.');
         }
-        $config = $container->get(\VuFind\Config\ConfigManagerInterface::class)->getConfigObject('config')->DOI;
+        $config = $container->get(\VuFind\Config\ConfigManagerInterface::class)->getConfigObject('config')->IdentifierLinks;
```

Comment:
To address the long-line style issue here and for backward compatibility (in case of outdated config files), I would suggest something like:


```suggestion
        $fullConfig = $container->get(\VuFind\Config\ConfigManagerInterface::class)->getConfigObject('config');
        $config = $fullConfig->IdentifierLinks ?? $fullConfig->DOI ?? null;
```